timer.py

```diff
@@ -0,0 +1,8 @@
+import time
+
+
+def timer(func, parameter):
+    start_time = time.time()
+    func(parameter)
+
+    print(f"It took {time.time() - start_time} seconds for the function to run")
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline